4.3/4.3.17/debian/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN useradd --system -m -d /plone -U -u 500 plone \
 
 COPY buildout.cfg /plone/instance/
 
-RUN buildDeps="wget gcc libc6-dev libpcre3-dev libssl-dev libxml2-dev libxslt1-dev libbz2-dev libjpeg62-turbo-dev libtiff5-dev libopenjp2-7-dev zlib1g-dev" \
+RUN buildDeps="build-essential wget gcc libc6-dev libpcre3-dev libssl-dev libxml2-dev libxslt1-dev libbz2-dev libjpeg62-turbo-dev libtiff5-dev libopenjp2-7-dev zlib1g-dev" \
```

Comment:
I don't think `build-essential` is the package you intended here: (from https://packages.debian.org/stretch/build-essential)

> If you do not plan to build Debian packages, you don't need this package.

What I would recommend instead is actually figuring out which libraries you're missing.  From what I can tell, the only things `build-essential` has that this list currently doesn't are `g++`, `make`, and `dpkg-dev`, so it shouldn't take much to narrow that down.

In testing, I found that it's `dpkg-dev` specifically that's necessary, due to the following bit from the `setup.py` of `Pillow`: (https://github.com/python-pillow/Pillow/blob/959db43d22b633519ac61ad6f58cbe63f8c1a1d0/setup.py#L740-L744)

```python
            try:
                ret = subprocess.call(['dpkg-architecture',
                                       '-qDEB_HOST_MULTIARCH'], stdout=fp)
            except Exception:
                return
```

(So, if `dpkg-architecture` fails, it will silently _not_ add the appropriate Debian architecture-specific library paths to the compiler's "library dirs" and "include dirs", which causes the compiler to be unable to find zlib.)

After replacing this with `dpkg-dev` instead, I was able to successfully build on arm64.  I would also recommend sorting this list to help ensure you don't list the same dependency twice, but that's purely a cosmetic suggestion (and doesn't really cause any ill effect to the build).

Reply:
+1 - Nice review